app/models/link_set.rb

```diff
@@ -1,42 +1,26 @@
 class LinkSet < ActiveRecord::Base
-  include Replaceable
-  include DefaultAttributes
-  include SymbolizeJSON
-
-  validate :links_are_valid
+  has_many :links
 
-  def links=(links_hash)
-    super(links_hash || {})
-  end
+  include Replaceable
 
-private
-  def self.query_keys
-    [:content_id]
-  end
+  def hashed_links
```

Comment:
This method has been introduced to display links as was done before by the json blob column that has been removed. Maybe I should rename it?